src/main/cli/settings.c

```diff
@@ -853,12 +853,11 @@ const clivalue_t valueTable[] = {
 #endif
 
 // PG_MOTOR_CONFIG
-    { "min_throttle",               VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { PWM_PULSE_MIN, PWM_PULSE_MAX }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, minthrottle) },
     { "max_throttle",               VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { PWM_PULSE_MIN, PWM_PULSE_MAX }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, maxthrottle) },
     { "min_command",                VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { PWM_PULSE_MIN, PWM_PULSE_MAX }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, mincommand) },
     { "motor_kv",                   VAR_UINT16 | HARDWARE_VALUE, .config.minmaxUnsigned = { 1, 40000 },                   PG_MOTOR_CONFIG, offsetof(motorConfig_t, kv) },
+    { PARAM_NAME_MOTOR_IDLE, VAR_UINT16  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 2000 },                            PG_MOTOR_CONFIG, offsetof(motorConfig_t, motorIdle) },
```

Comment:
`min_throttle` was able to go from 750 to 2250. That's even more than the suggested limit from 0 to 10.000
```suggestion
    { PARAM_NAME_MOTOR_IDLE, VAR_UINT16  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 10000 },                            PG_MOTOR_CONFIG, offsetof(motorConfig_t, motorIdle) },
```

Reply:
The effective min_throttle can still go from 750 (plus the idle value / 10) to 2250.
Wit the max configurable idle of 2000, we are able to add 20%, effectively, to the motor drive, above min_command.
For example, for an analog esc with min_command configured to 750, a motorIdle setting of 500 will result in a motor value of 800 being applied at idle.
That's exactly what a user would want, because the normal configuration of an analog ESC is to set min_command just below spin point.  Not a huge way below.
If the user has a very low quality analog signal, they may have to set min_command some distance below the point at which the motors start, in order to avoid twitching.
In that case they may need to increase the motorIdle value to a higher than typical value.  
However I think these cases are very unusual.  Analog ESCs can usually be calibrated to respond to the normal 1020-2000 range.
There was never any point setting min_throttle to 750, it always had to be set above min_command by the amount required to achieve a suitable idle.